docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -0,0 +1,65 @@
+# Using devices with Clear Containers
+
+Host devices can be passed to a container with Docker using `--device` option
+with docker run. An example of passing a device with docker is given below:
+```
+$ sudo docker run -it --device=/dev/sdb debian
+```
+Clear Containers has support for passing the following devices to the
+container with `--device`.
+
+## Virtual Function I/O (VFIO) Devices (PCI Passthrough)
+
+The VFIO driver is a framework for exposing direct device access to userspace, 
+in a secure, Input–Output Memory Management Unit (IOMMU) protected environment.
+A device can be directly passed to a Clear Containers container using VFIO. 
+
+For information on VFIO, please refer to:
+ - https://www.kernel.org/doc/Documentation/vfio.txt
+ - https://www.linux-kvm.org/images/b/b4/2012-forum-VFIO.pdf
+
+Devices can pe passed to a Clear Container container using VFIO
```

Comment:
Lines 21-22 rewrite: "The following example uses VFIO to pass a device to a Clear Container container:"